src/d3a/d3a_core/simulation.py

```diff
@@ -47,7 +47,6 @@
 from d3a.d3a_core.exceptions import D3AException
 from d3a.models.area.event_deserializer import deserialize_events_to_areas
 from d3a.d3a_core.live_events import LiveEvents
-from d3a.d3a_core.singletons import aggregator
```

Comment:
Please remove the de declaration of `aggregator` in the src/d3a/d3a_core/singletons.py (or remove the whole file)

Reply:
sorry, you did already.